### PR TITLE
Fixed patient encounter gui

### LINKED ISSUE #3017: Collapse/expand are not working on the Encounter gui
### Describe the bug

Collapse/expand are not working on the Encounter gui

### To Reproduce

1. Go to the patient encounter gui
1. Create a SOAP note there and save it
1. Note that the expand/collapse is not working correctly (the labels change but the data fields are not expanding or collapsing)

image

Nothing is showing up in the js console.

### Expected behavior

The data fields should collapse/expand. I am willing to bet there is a simple js hide/show issue going on.

### Client configuration

- Browser: Firefox 72.0.2
- Operating system: Ubuntu 16.04

### PR DESCRIPTION
**Pull request by adad20**
Fixes #3017 

#### Short description of what this resolves:
Fixed the working of the expand / collapse button in encounter gui

#### Changes proposed in this pull request:

-Changed the UI
-Used BS4 JS function instead of expandcollapse function

### CODE REVIEW

File: interface/patient_file/encounter/forms.php [hunk 1]
```diff
@@ -770,8 +757,7 @@ function myGetRegistered($state = "1", $limit = "unlimited", $offset = "0")
 <?php if ($GLOBALS['enable_follow_up_encounters']) { ?>
     <a href='#' class='btn btn-primary' onclick='return createFollowUpEncounter()'><?php echo xlt('Create follow-up encounter') ?></a>
 <?php } ?>
-&nbsp;&nbsp;&nbsp;<a href="#" onClick='expandcollapse("expand");' style="font-size:80%;"><?php echo xlt('Expand All'); ?></a>
-&nbsp;&nbsp;&nbsp;<a  style="font-size: 80%;" href="#" onClick='expandcollapse("collapse");'><?php echo xlt('Collapse All'); ?></a>
+&nbsp;&nbsp;&nbsp;<button type="button" class="btn btn-primary btn-sm" data-toggle="collapse" data-target="#demo"><?php echo xlt('Expand / Collapse'); ?></button>
```
Comment (im-Amitto) [inline interface/patient_file/encounter/forms.php]: can you please replace those `&nbsp`  with padding/margin. I know it's not part of your PR but they smell bad.

**Comment by bradymiller** [comment]
image

I like the idea of removing the individual expand/collapse for each form and just making it a simple expand/collapse for all of them. Markedly simplifies code also.

In the screenshot, note it's tough to tell the separation of the form and what form the signed elements correspond to. Need to add likely add margin to bottom of forms and move up the sign log elements.

**Comment by bradymiller** [comment]
Also, lets have it Expand by default.

**Comment by im-Amitto** [comment]
@adad20 I'm not sure it this is the expected behaviour.
https://www.loom.com/share/cee19b64e28f46879cb3ff0839c57771

**Comment by adad20** [comment]
@im-Amitto Its working fine on my system 
[https://www.loom.com/share/46428b1bce8a454786d9aa5587a04e44](url)

**Comment by im-Amitto** [comment]
@adad20 create an eye exam encounter and you will see what i'm talking about.

**Comment by bradymiller** [comment]
hi @adad20 and @im-Amitto , The eye form was causing issues to this gui even prior to this fix. So plan to bring this in (gonna do a quick test first), and then we can make an issue regarding the eye form.

**Comment by bradymiller** [comment]
hi @adad20 , Testing well (except for the prior eye form bug) and bringing this into codebase. Thanks for the code improvements! -brady

**Comment by im-Amitto** [comment]
Oh, @bradymiller  I did't knew that eye form was't related to the collapse/expand feature. Anyway nice work @adad20 :smile: .

**Pull request merged**